### ISSUE TITLE
Description of how to encode configuration is hard to follow or incorrect.

### ISSUE THREAD

**Issue by derrickburns** [pomerium/pomerium#685]
I have been using `pomerium/pomerium:v0.6.2` for some time with no problem using the Helm chart 5.0.3.

I am trying to migrate to `pomerium/pomerium:v0.7.5` with Helm chart 8.5.4.

I pass most of my configuration via a Secret that I create offline, including these fields:
```
stringData:
    config.yaml: |
        cookie_secret:            # base64 encoded string
        idp_client_id:            # unencoded string
        idp_client_secret:        # unencoded string
        idp_service_account:      # base64 encoding of serialized JSON
        shared_secret:            # base64 encoded string
```

First, the documentation does **not** say that `idp_service_account` must be base64 encoded.  Through experimentation, I have determined that it does. That is the bug I am reporting. 

Second, even with this change, I cannot log in to my use Google as IDP.  This worked before with the same values for `idp_service_account`.

I get this error upon login attempt:

```
oauth callback : error redeeming authenticate code: internal/identity: token exchange failed: oauth2: cannot fetch token: 401 Unauthorized\nResponse
```

What do I need to do?

**Comment by desimone** [comment]
Hey @derrickburns ,

Sorry to hear you are having issues with upgrading to v0.7.5.

> First, the documentation does not say that idp_service_account must be base64 encoded. Through experimentation, I have determined that it does. That is the bug I am reporting.

```sh
IDP_SERVICE_ACCOUNT="zzzz" # output of `cat service-account-key.json | base64`
```
https://www.pomerium.io/docs/identity-providers/google.html
I think this can be called out better (but, unfortunately, it can vary from identity provider to identity  provider. For example, Okta also uses service account, and in that case the input is not base64 encoded.

**Comment by desimone** [comment]
>First, the documentation does not say that idp_service_account must be base64 encoded. Through experimentation, I have determined that it does. That is the bug I am reporting.


> I think this can be called out better (but, unfortunately, it can vary from identity provider to identity provider

This should at least be somewhat better as of v0.10.0 all the identity providers wrap their `idp_service_account` in base64 encoded JSON.
